test/helpers/position_assertions.cc

```diff
@@ -42,6 +42,14 @@ const UnorderedSet<string> ignoredAssertionLabels = {"typed", "TODO", "lineariza
 constexpr string_view NOTHING_LABEL = "(nothing)"sv;
 constexpr string_view NULL_LABEL = "null"sv;
 
+bool startsWith(const std::string &str, const std::string &prefix) {
```

Comment:
Why not `absl::StartsWith`/`absl::EndsWith`?

Reply:
:mindblown:

Reply:
I even looked to see if these existed and didn’t find them

Reply:
I've been there. There's a lot of nice stuff in absl, but not all of it is easy to find!